src/containers/CancelButton.js

```diff
@@ -6,7 +6,7 @@ import { WarningModal } from '../components/Modals/WarningModal';
 
 import { useDispatch } from 'react-redux';
 import { cancelPlaybookRuns } from '../actions';
-import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/esm/actions';
```

Comment:
No need to have full import, you have `frontend-components-notifications` import mapper in `babel.config.js`